IBM_CLOUD_DEPLOY.md

```diff
@@ -0,0 +1,29 @@
+# Deploying to IBM Cloud
+
+1. Create a cluster using the [IBM Cloud Container Service](https://console.bluemix.net/containers-kubernetes/clusters
+). For this Code Pattern we will only need to use the Free tier which gives us access to a Kubernetes cluster with 1 worker node. (Could take anywhere from 10 minutes to half an hour.)
+
+![Create Cluster](doc/source/images/create-cluster.png)
+
+2. (Optional Step) The instructions to install the command line tools are optional and not required to deploy the model using this tutorial. You may choose to install them while waiting for the cluster to be created, if you wish to do so.
+
+3. On the IBM Dashboard select `Worker Nodes` and copy the `Public IP Address` for the node.
+
+![Worker IP](doc/source/images/worker-public-ip.png)
+
+4. Open the `Kubernetes Dashboard`. Hit the `+ Create` button in the top right corner. Go to `Create an app`.
+   Fill in the following fields:
+   ```
+    App name: <Select a name>
+    Container image: codait/max-image-caption-generator:latest
+    Number of pods: 1
+    Service: External - Port: 5000, Target Port: 5000
```

Comment:
I think moving `Port ...` onto the next line would be more consistent with the page

Reply:
Also you should mention clicking `Deploy` at the end of this step